src/main/resources/org/sonar/l10n/checkstyle/rules/checkstyle/com.puppycrawl.tools.checkstyle.checks.AvoidEscapedUnicodeCharactersCheck.html

```diff
@@ -1,8 +1,8 @@
 <div >
-<h2><a name="AvoidEscapedUnicodeCharacters"></a>AvoidEscapedUnicodeCharacters</h2>
+<h2><a id="AvoidEscapedUnicodeCharacters"></a>AvoidEscapedUnicodeCharacters</h2>
```

Comment:
@romani I feel this change broke the original reason for this tag.
This code was similar to what the generated source of our xdocs look like.
`name` attribute of `a` allows you to place links to these sections similar to http://checkstyle.sourceforge.net/config_filters.html#SuppressionXpathSingleFilter .
Switching it to `id` serves absolutely no purpose if you aren't referencing this tag with javascript or css because `id` is just a unique identifier to identify this tag out of all of the ones on the page.

IMO, code like this should be reverted, or removed completely.

Reply:
You can use Id after `#` , links will work.
Content of such pages are so outdated, so investing time in them not is valuable. I just tried to make CI green.
Doc in plugin should have only short description and link to our site.

Looks like we have the same violations in main project, so

Reply:
Looks like you are right, it changed for new HTML versions (HTML 5).
https://stackoverflow.com/a/484781/1016482
Old browsers won't support `id`.